cognitedata/reveal
# feat(react-components): rulebased outputs bottom panel and radio button

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:

https://cognitedata.atlassian.net/browse/BND3D-4192
https://cognitedata.atlassian.net/browse/BND3D-4107

https://cognitedata.atlassian.net/browse/

## Description :pencil:
Introduce the bottom panel to show the output labels for the selected RuleSet.
Also change the switcher to a radio button for the RuleSet selection

## Test instructions :information_source:

1. Select a RuleSet in the Search UI
2. Observe the bottom left panel with the output labels


## Checklist :ballot_box_with_check:
